dcc-portal-ui/config/webpack.config.dev.js

```diff
@@ -5,7 +5,7 @@ var CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
 var paths = require('./paths');
 
 module.exports = {
-  devtool: 'cheap-module-eval-source-map',
+  devtool: 'source-map',
```

Comment:
Please don't commit this
(didn't we make something on a branch of yours that determines which sourcemap mode to use based on environment variables?)